support/cas-server-support-inwebo-mfa/src/main/resources/webflow/mfa-inwebo/mfa-inwebo-webflow.xml

```diff
@@ -0,0 +1,64 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<flow xmlns="http://www.springframework.org/schema/webflow"
+      xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+      xsi:schemaLocation="http://www.springframework.org/schema/webflow
+                          http://www.springframework.org/schema/webflow/spring-webflow.xsd">
+
+    <on-start>
```

Comment:
All webflow activity should be constructed dynamically in webflow-configurer classes with no XML whatsoever, except perhaps one end-state. The current webflow module cannot be constructed with an XML configuration file, so we'd include one state in the configuration to allow for the initialization step, and build everything dynamically in the webflow configurer (which you can of course unit test unlike xml)

Reply:
Currently, it works with the XML webflow. Or maybe there is a side-effect I didn't side.

I will change that too for consistency at the end of the rework.

Reply:
The main driver is that you can test the webflow configurer with unit tests, and its behavior can be configured by an adopter without having to override xml config. In the very near future, the xml-base for all webflow will also be removed so this would be more in line with that as well.

Reply:
Done